Add PPT for PUBG (PC Wiki)

Hesketh2 opened the pull request:
## Summary
This is the port of the #2138 (/Custom PPT from PUBG Mobile) (includes any subsequent changes that is done after this, because I paste from the pubgm wiki just today)

## How did you test this change?
since {{TeamPrizePool}} hasnt been used in past events before this, I converted it on past events

https://liquipedia.net/pubg/PUBG_Global_Championship/2022#Prize_Pool

Rathoz commented:
Is /Legacy planned as well including legacy conversion?

hjpalpha reviewed (approved):
LGTM

Hesketh2 commented:
> Is /Legacy planned as well including legacy conversion?

Yeah I do planned to convert the old prize pool start template to use the standard modules, I havent done that yet.
Over at PUBGM theres #2137 that was left very long ago

hjpalpha commented:
> > Is /Legacy planned as well including legacy conversion?
> 
> Update: So now I've moved the old prize pool start template to using Module:PrizePool/Legacy now but the module is still a commons (no custom yet). How can I determine if a wiki needs a Legacy Custom since I think if not having Legacy Custom a more viable solution?
> 
> (like, idk how much Legacy/Custom is needed for PUBG over a commons base module)

PrizePool/Legacy/Custom should be used in the invoke, not PrizePool/Legacy
(there is a default PrizePool/Legacy/Custom on commons)

Hesketh2 commented:
> > > Is /Legacy planned as well including legacy conversion?
> > 
> > 
> > Update: So now I've moved the old prize pool start template to using Module:PrizePool/Legacy now but the module is still a commons (no custom yet). How can I determine if a wiki needs a Legacy Custom since I think if not having Legacy Custom a more viable solution?
> > (like, idk how much Legacy/Custom is needed for PUBG over a commons base module)
> 
> PrizePool/Legacy/Custom should be used in the invoke, not PrizePool/Legacy (there is a default PrizePool/Legacy/Custom on commons) we need to check what works with that and what doesn't and according to that we need to build one or the wiki or not since you are adding duo support via the custom opponentDisplay module you most certainly also need a ppt legacy custom on the wiki

My bad, I mean the Prizepool/Legacy/Custom that is the default commons is the one that is currently  used, I didnt fully written and it turns out I misexplained it as another module.

![image](https://user-images.githubusercontent.com/88981446/219301394-814a4c5f-a5f8-4c45-b2a6-3867ed95cce6.png)

the current way of how I do the duos is I currently have the **{{prize pool start/slot/end duos}}** called **Module:PrizePool/Fortnite/next** which is the Fortnite prize module on commons since that one works with duo input already (and later the results table with the OpponentDisplay modules). 

It's why that led to my idea if PUBG can go ahead without needed of Local PrizePool/Legacy/Custom since
- the old prize used **|points=seed** to indicate qualification
- pubg didnt use **usdprize2 or localprize2**
- The solos mode tournament I have converted those to {{SoloPrizePool}} already so the Legacy/Custom now only used under team-based prize pool only

hjpalpha commented:
depends on #2595 for duo support
if smw storage for duos is wanted this PR needs adjusting accordingly

Hesketh2 commented:
update so the final decision is Prizepool/Legacy/Custom is not needed for this wiki. So only this one to be made
